config/routes.rb

```diff
@@ -1,10 +1,14 @@
 require 'sidekiq/web'
 
 Rails.application.routes.draw do
+  get 'sessions/new'
+
   mount Sidekiq::Web => '/sidekiq'
 
   scope module: 'admin' do
     get 'admin/contests/:contest_id/scoreboard' => 'scoreboard#show', as: :admin_contest_scoreboard
+    post   'admin/login',   to: 'sessions#create'
+    delete 'admin/logout',  to: 'sessions#destroy'
```

Comment:
if you want to name these two routes, add an `as:` key .... otherwise, this is more or less equivalant to `resources :admin, only: [:create, :destroy]`